### PR TITLE
Link to consumer groups Grafana dashboard

### LINKED ISSUE #32: Link to consumer group dashboard from streams apps and connectors

### LINKED ISSUE #37: Link to Consumer groups Grafana dashboard

### PR DESCRIPTION
**Pull request by disrupted**
Closes #32

- [x] Update Readme
- [x] streaming apps
- [x] connectors

### CODE REVIEW

**Review by philipp94831 — CHANGES_REQUESTED** [review]
What about connectors?

**Comment by disrupted** [comment]
Added connectors and also a better test that mocks the consumer group annotation.

File: backend/streams_explorer/defaultlinker.py [hunk 1]
```diff
@@ -19,9 +19,22 @@ def __init__(self):
...
+                name="Consumer Group Monitoring",
+                value="grafana",
+                type=NodeInfoType.LINK,
+            ),
+            NodeInfoListItem(
+                name="Kibana Logs", value="kibanalogs", type=NodeInfoType.LINK
+            ),
+        ]
+        self.connector_info = [
+            NodeInfoListItem(
```
Comment (philipp94831) [inline backend/streams_explorer/defaultlinker.py:32]: Can't you reuse the same NodeInfoListItem from streaming_app_info?

Comment (disrupted) [reply backend/streams_explorer/defaultlinker.py:32]: I can change it of course. imo it looks a little less structured and readable though, what do you think?
```python
grafana_consumer_link = NodeInfoListItem(
    name="Consumer Group Monitoring", value="grafana", type=NodeInfoType.LINK
)
self.topic_info = [
    NodeInfoListItem(
        name="Topic Monitoring", value="grafana", type=NodeInfoType.LINK
    ),
    NodeInfoListItem(
        name="Message Viewer", value="akhq", type=NodeInfoType.LINK
    ),
]
self.streaming_app_info = [
    grafana_consumer_link,
    NodeInfoListItem(
        name="Kibana Logs", value="kibanalogs", type=NodeInfoType.LINK
    ),
]
self.connector_info = [grafana_consumer_link]
self.sink_source_info = {
    "elasticsearch-index": [
        NodeInfoListItem(name="Kibana", value="", type=NodeInfoType.LINK)
    ]
}
```

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**